<ui version="4.0" >
 <class>jServiceBrowserClass</class>
 <widget class="QWidget" name="jServiceBrowserClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>jServiceBrowser</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_8" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="maximumSize" >
        <size>
         <width>51</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Server:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="serviceServer" >
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="goButton" >
       <property name="text" >
        <string>></string>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QTreeWidget" name="servicesWidget" >
     <property name="verticalScrollBarPolicy" >
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="showDropIndicator" stdset="0" >
      <bool>true</bool>
     </property>
     <property name="columnCount" >
      <number>1</number>
     </property>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_5" >
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>jServiceBrowserClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>376</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel" >
     <x>261</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
